cmd/ctr/overlaybd_conv.go

```diff
@@ -95,13 +113,44 @@ var convertCommand = cli.Command{
 			Usage: "data base config string used for layer deduplication",
 			Value: "",
 		},
+		cli.BoolFlag{
+			Name:   "push-artifact",
+			Usage:  "convert to OCI Artifact, add original manifest as referrer, and push to specified registry",
+			Hidden: false,
+		},
+		cli.StringFlag{
+			Name:  "username",
+			Usage: "username of artifact destination registry(required), used for authenticating to destination artifact registry",
+			Value: "",
+		},
+		cli.StringFlag{
+			Name:  "password",
+			Usage: "password of artifact destination registry(required), used for authenticating to destination artifact registry",
+			Value: "",
+		},
```

Comment:
Can change/add this to the traditional `--user username:password` flag, example is here:

https://github.com/akashsinghal/accelerated-container-image/blob/main/cmd/ctr/rpull.go#L37

Reply:
Switching to using the containerd `--user username:password` flag